Compilation issue

The recent merge has left a compilation problem

```
error[E0432]: unresolved import `crate::self_signed_certs`
 --> server/src/discovery.rs:1:12
  |
1 | use crate::self_signed_certs::Protocol;
  |            ^^^^^^^^^^^^^^^^^ could not find `self_signed_certs` in the crate root
```

--- a/server/server/src/discovery.rs
+++ b/server/server/src/discovery.rs
@@ -1,4 +1,4 @@
-use crate::self_signed_certs::Protocol;
+use crate::certs::Protocol;
 use local_ip_address::list_afinet_netifas;
 use serde::Serialize;
 use service::settings::ServerSettings;

